CONTRIBUTING.rst

```diff
@@ -27,6 +27,10 @@ Contributing guidelines
 * Ensure your code is tested and documented. Patches without 100% test coverage
   or documentation will not be accepted.
 * Write `good commit messages`_.
+* Once you are ready with your changes and before you submit it for review to
+  make sure the API docs are updated with your changes, please run::
+
+    sphinx-apidoc -f -o docs/_autodoc tls/ --private
```

Comment:
What if we use a tox testenv to provide this to the user so they can do

```
tox -e gendoc
```

And that way they don't have to worry about installing sphinx in a virtualenv or however else they might do it.